database/dbGenerator.js

```diff
@@ -1,26 +1,12 @@
+const _ = require('lodash');
 const faker = require('faker');
-const { urlPrefix } = require('./imageUrls');
-const { japanUrls } = require('./imageUrls');
-const { italyUrls } = require('./imageUrls');
-const { usUrls } = require('./imageUrls');
-const { singaporeUrls } = require('./imageUrls');
-const { bathroomUrls } = require('./imageUrls');
-const { bedroomUrls } = require('./imageUrls');
-const { kitchenUrls } = require('./imageUrls');
-const { livingRoomUrls } = require('./imageUrls');
-const { otherUrls } = require('./imageUrls');
+const imageUrls = require('./imageUrls');
 
 
-// Function that generates a random number
-const randomNumber = (min, max) => {
-  const num = Math.floor(Math.random() * (max - min + 1)) + min;
-  return num;
-};
-
 // Function that checks if photo has already been selected
 const photoIdx = (urlCategory, selectedPhotos) => {
   // Randomly generate a number between 0 and the amount of photos minus 1
-  const currPhotoIdx = randomNumber(0, urlCategory.length - 1);
+  const currPhotoIdx = _.random(0, urlCategory.length - 1);
```

Comment:
@aku2nnya Nice re-factor! This is also supported through faker in case it is helpful! (https://stackoverflow.com/questions/32010910/faker-js-random-number-between-2-values)